README.md

```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
```

Comment:
Nice addition of the image, plus a good idea to store it in a separate images folder. It pays to keep your repos organized so everything isn't scattered in the same main folder